text/src/autogluon/text/automm/data/trivial_augmenter.py

```diff
@@ -0,0 +1,252 @@
+"""
+This file implements TrivialAugment.(https://arxiv.org/abs/2103.10158) We extend it for multi-modality setting. 
+
+Code is partically adapted from its official implementation https://github.com/automl/trivialaugment
+"""
+
+import random
+from selectors import EpollSelector
+from PIL import ImageOps, ImageEnhance, ImageFilter, Image, ImageDraw
+import nlpaug.augmenter.word as naw
+from .utils import InsertPunctuation
+import nltk
+
+
+def scale_parameter(level, maxval, type):
+    """Helper function to scale `val` between 0 and maxval .
```

Comment:
Need to adjust the docstrings' format. Separate triple quotes and texts in different lines.